scripts/kani-regression.sh

```diff
@@ -30,6 +30,7 @@ PYTHONPATH=${SCRIPT_DIR} python3 -m unittest ${SCRIPT_DIR}/test_cbmc_json_parser
 
 # Build all packages in the workspace
 cargo build --workspace
+${SCRIPT_DIR}/refresh-kani-proptest.sh
```

Comment:
Can we make this part if the cargo build?

Reply:
As discussed in the mtg:  Cargo build will not be possible because the kani-compiler binary needs to be ready at this point. I will make it integrate with cargo kani instead.

Reply:
Turns out this refresh was not needed. The extra messages were confused with  https://github.com/model-checking/kani/issues/1478. Removed as of 44a40742a9b13887ef14666a41a6e7269048127f